src/v2/Apps/Onboarding/useOnboardingContext.tsx

```diff
@@ -10,38 +10,40 @@ import {
   SetStateAction,
 } from "react"
 import { WorkflowEngine } from "v2/Utils/WorkflowEngine"
-import { useEngine } from "./config"
+import { useConfig } from "./config"
 
-export type Context = {
-  answer: string | null
+type Answer = string | null
+
+export type Basis = {
+  answer: Answer
 }
 
-export const DEFAULT_CONTEXT: Context = {
+export const DEFAULT_BASIS: Basis = {
   answer: null,
 }
 
-type AnswersState = [] | [string] | [string, string]
-
 const OnboardingContext = createContext<{
-  answers: AnswersState
-  context: React.RefObject<Context>
+  answers: [Answer, Answer]
+  basis: React.RefObject<Basis>
   current: string
-  engine: WorkflowEngine
+  workflowEngine: WorkflowEngine
   next(): void
   onDone(): void
   progress: number
-  setAnswers: Dispatch<SetStateAction<AnswersState>>
-  setContext: (updatedContext: Partial<Context>) => React.RefObject<Context>
+  setAnswerOne: Dispatch<SetStateAction<Answer>>
+  setAnswerTwo: Dispatch<SetStateAction<Answer>>
```

Comment:
If there were three multiple choice options, would we add another option here?

Reply:
This whole thing is gonna change so I'm not gonna over think this; but something like that. Or we might model it differently depending.